#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_p1_pl_shebang.dpatch by Marc Haber <mh+debian-packages@zugschlus.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix shebang of perl interpreter

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' pkg-nagios3~/p1.pl pkg-nagios3/p1.pl
--- pkg-nagios3~/p1.pl	2012-05-07 20:43:10.000000000 +0200
+++ pkg-nagios3/p1.pl	2012-05-07 20:48:31.118953001 +0200
@@ -1,4 +1,6 @@
- package Embed::Persistent;
+#!/usr/bin/perl
+
+package Embed::Persistent;
 
 # p1.pl for Nagios
 
